scripts/framework-applications/generate-successful-suppliers-for-framework-csv.py

```diff
@@ -1,15 +1,18 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-Generate a CSV listing all suppliers who were successfully awarded onto a framework with the lots that they were
-successful on. This CSV is published as an update communication through the Digital Marketplace when the framework
-is awarded (i.e. when standstill ends).
+Generates a CSV listing all suppliers who have had services published to a framework, and the lots that they were
+successful on. This report is useful for tallying the results of the framework during standstill (i.e. just before the
+framework goes live).
+
```

Comment:
We should add a line to the docstring on where the data goes, or make it so that it outputs to standard out.

Reply:
Good idea. Don't want to faff with this script too much so have added info to the docstring.